sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -199,6 +210,37 @@ private void mergeOS(final @NotNull SentryBaseEvent event) {
     }
   }
 
```

Comment:
so I'm thinking querying DeviceInfoUtil is probably too much work for this, shall we maybe just directly use the const values for brand, model and family from `android.os.Build`? Same for the OS name and version, those are also const values. Just trying to minimize impact since logs can happen frequent and originate from the main thread.